addons/payment/views/payment_templates.xml

```diff
@@ -63,6 +63,12 @@
                                        t-att-checked="acquirers_count==1 and pms_count==0 or acquirers[0] == acq"/>
                             </t>
                             <span class="payment_option_name" t-esc="acq.name"/>
+                            <t t-if="acq_extra_fees and acq_extra_fees.get(acq)">
+                                <span class="badge badge-primary"> + <t t-esc="acq_extra_fees[acq]" t-options='{"widget": "monetary", "display_currency": acq_extra_fees["currency_id"]}'/> Fee </span>
+                            </t>
+                            <t t-elif="acq.fees_active">
```

Comment:
This `elif` case was a bit mysterious when reading at first, can you confirm that it is needed for:

- Either special controller that would not set `acq_extra_fees` in `values` dict to render like the `/website_payment/pay' (that probably can't do it anyway since no currency).
- Either for `payment.acquirer` that activated `fees` but did not override `XX_compute_fees`, but such a case do not exists in standard Odoo code.

Reply:
Hello @rdeodoo , 
As discussed with FGI, we need it in the case where,
 - extra_fees is active and either the method to calculate extra_fees is not implemented or the extra fees is zero. 

cc @mba-odoo